### ISSUE TITLE
Run classification on the 3D AEM dataset plus other supplementary dataset

### ISSUE THREAD

**Issue by niketchhajed** [GeoscienceAustralia/uncover-ml#42]
After the training dataset has been cleaned, normalised and scaled, the next step is to use different classification algorithms using an open source ML library like scikit-learn using different parameters for learning rate, initial co-efficients, and several other configurations sensitive to the algorithm.

**Comment by niketchhajed** [comment]
Some initial results were generated using scikit-learn and shared with John Wilford. The analysis of these results points to the possibility of increasing the coverage of labeled training dataset.

**Comment by niketchhajed** [comment]
The final results were plotted and shared and we can see acceptable results as per discussion with John.
